src/index.test.ts

```diff
@@ -1,8 +1,49 @@
 import Geckoboard from './index';
+import { MockAgent, setGlobalDispatcher } from 'undici';
 
 describe('Geckoboard', () => {
   it('has the version set correctly', () => {
     const gb = new Geckoboard('API_KEY');
     expect(gb.version).toBe('2.0.0');
   });
+
+  it('can ping the api server', async () => {
+    const mockAgent = new MockAgent();
+    setGlobalDispatcher(mockAgent);
+
+    const mockPool = mockAgent.get('https://api.geckoboard.com');
+    mockPool
+      .intercept({
+        path: '/',
+        headers: { Authorization: `Basic ${btoa('API_KEY:')}` },
+      })
```

Comment:
🏄 not vital, but is there a way to have this record the calls made and then assert on them at the end of the test?